### PR TITLE
move files from nwbconversiontools

### PR DESCRIPTION
**Pull request by Saksham20**
NWBConverter child classes, and conversion scripts are moved here.

### CODE REVIEW

**Comment by bendichter** [comment]
please use tqdm for long scripts

File: giocomo_lab_to_nwb/ophys/giocomovrdatainterface.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+        # adding stimulus:
+        for inp_kwargs in self.stimulus_args:
+            if inp_kwargs['name'] not in nwbfile.stimulus:
+                inp_kwargs.update(starting_time=start_time, rate=rate, data=self.data_frame[inp_kwargs['name']].to_numpy())
+                nwbfile.add_stimulus(TimeSeries(**inp_kwargs))
+
+
+subject_info = \
+    {'4139265.3': {'mouse #': '4139265.3', 'species': 'mus musculus', 'ID': 'R1', 'DOB': '2018-11-07 00:00:00',
```
Comment (bendichter) [inline giocomo_lab_to_nwb/ophys/giocomovrdatainterface.py]:
```suggestion
    {'4139265.3': {'mouse #': '4139265.3', 'species': 'Mus musculus', 'ID': 'R1', 'DOB': '2018-11-07 00:00:00',
```

File: giocomo_lab_to_nwb/ophys/giocomovrdatainterface.py [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+                   'VIRUS': 'AAV1-CAG-FLEX-GCaMP6f-WPRE', 'cannula implant date': '2019-07-10 00:00:00',
+                   'weight at time of implant': '22.3 g'},
+     '4222169.2': {'mouse #': '4222169.2', 'species': 'mus musculus', 'ID': 'FD3', 'DOB': '2019-03-03 00:00:00',
+                   'genotype': 'CaMKII-cre hemizygous', 'sex': 'FEMALE', 'virus injection date': '2019-07-10 00:00:00',
+                   'VIRUS': 'AAV1-CAG-FLEX-GCaMP6f-WPRE', 'cannula implant date': '2019-07-10 00:00:00',
+                   'weight at time of implant': '18.7 g'},
+     '4222169.4': {'mouse #': '4222169.4', 'species': 'mus musculus', 'ID': 'FD4', 'DOB': '2019-03-31 00:00:00',
+                   'genotype': 'CaMKII-cre hemizygous', 'sex': 'FEMALE', 'virus injection date': '2019-07-12 00:00:00',
+                   'VIRUS': 'AAV1-CAG-FLEX-GCaMP6f-WPRE', 'cannula implant date': '2019-07-12 00:00:00',
+                   'weight at time of implant': '21.7 g'}}
```
Comment (bendichter) [inline giocomo_lab_to_nwb/ophys/giocomovrdatainterface.py]: this should be included as a csv or excel file

File: giocomo_lab_to_nwb/ophys/giocomovrdatainterface.py [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+        date = self.file_path.parents[1].name
+        subject_num = self.file_path.parents[2].name
+        session_desc = self.file_path.stem
+        subject_details = subject_info[subject_num]
+        metadata = dict(
+            NWBFile=dict(
+                session_description=session_desc,
+                identifier=str(uuid.uuid4()),
+                session_start_time=datetime.strptime(date, "%m_%d_%Y"),
+                experiment_description=exp_desc
```
Comment (bendichter) [inline giocomo_lab_to_nwb/ophys/giocomovrdatainterface.py]: add virus from the subject data. Also add canula implant date under the `surgery` arg

File: giocomo_lab_to_nwb/ophys/giocomovrdatainterface.py [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+        exp_desc = self.file_path.parents[0].name
+        date = self.file_path.parents[1].name
+        subject_num = self.file_path.parents[2].name
+        session_desc = self.file_path.stem
+        subject_details = subject_info[subject_num]
+        metadata = dict(
+            NWBFile=dict(
+                session_description=session_desc,
+                identifier=str(uuid.uuid4()),
+                session_start_time=datetime.strptime(date, "%m_%d_%Y"),
```
Comment (bendichter) [inline giocomo_lab_to_nwb/ophys/giocomovrdatainterface.py]: add PT timezone

File: giocomo_lab_to_nwb/ophys/giocomoconverter.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
+from nwb_conversion_tools import NWBConverter, SbxImagingInterface, Suite2pSegmentationInterface
+from .giocomovrdatainterface import GiocomoVRInterface
+from pathlib import Path
+import warnings
```
Comment (bendichter) [inline giocomo_lab_to_nwb/ophys/giocomoconverter.py:4]: please optimize imports for all .py files

File: giocomo_lab_to_nwb/ophys/giocomovrdatainterface.py (hunk 2 above)
Comment (Saksham20) [reply giocomo_lab_to_nwb/ophys/giocomovrdatainterface.py]: i've included a json file instead, makes it easier to look up a subject id specific info as a dicts keys

**Pull request merged**